packages/jest-jasmine2/src/index.js

```diff
@@ -263,12 +263,15 @@ function jasmine2(config, environment, moduleLoader, testPath) {
   env.addReporter(reporter);
   moduleLoader.requireModule(testPath);
   env.execute();
-  env.snapshotState.snapshot.save();
+  const operationResult = env.snapshotState.snapshot.save();
```

Comment:
pretty much everything is an operation or instruction, so adding it to the variable name isn't really helpful. Maybe just call it `saveState` or `status` or `result`?